spec/controllers/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+describe BookSuggestionsController do
+  describe 'POST #create' do
+    subject(:http_response) { post :create, params: { book_suggestion: book_suggestion } }
+
+    context 'when posting a new book suggestion' do
+      let(:book_suggestion) do
```

Comment:
:thinking: you can use FactoryBot.attributes_for(:book_suggestion) to do this :wink: [chek it](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-factories)